CONTRIBUTING.md

```diff
@@ -4,25 +4,32 @@
 
 Feel free to raise an issue if you have a question, an enhancement, or a bug report.
 
-Use the issue search functionality to search all **_opened and closed_** issues before raising a new issue. If you raise a duplicate issue, you're just creating noise for everyone watching this repo.
+Use the issue search functionality to search all **_opened and closed_** [issues](https://github.com/typescript-eslint/typescript-eslint/issues) before raising a new issue. If you raise a duplicate issue, you're just creating noise for everyone watching this repo.
 
 Before raising a bug, ensure you are using the latest version of our packages. We release every week, so there's a good chance your issue might have already been fixed.
 
-Finally, when raising a new issue, please fill out the issue template - **_please don't skip sections_**.
+Finally, when raising a new issue, please fill out the [issue template](https://github.com/typescript-eslint/typescript-eslint/issues/new/choose) - **_please don't skip sections_**.
 
 Please provide **_as much information as possible_**. This project is maintained by volunteers, so the more information you provide, the less likely we will have to waste everyone's time in asking you for more information.
 
-If you have a particularly complex issue - consider creating a small, self-contained reproduction repo. This will help you in figuring out the exact problem, and will help us in reproducing and diagnosing the bug.
+If you have a particularly complex issue - consider creating a small, self-contained reproduction repo or use our [playground](https://typescript-eslint.io/play). This will help you in figuring out the exact problem, and will help us in reproducing and diagnosing the bug.
 
 **_Help us to help you_**
 
+## Questions and requests for support
+
+Questions and requests for support should not be opened as issues and should be handled in the following ways:
+
+- Ask a question on [StackOverflow](https://stackoverflow.com/questions/tagged/typescript-eslint) using the `typescript-eslint` tag.
+- Join our [Discord server](https://discord.com/invite/??).
```

Comment:
Indeed, we don't have one. We're not paid enough to deal with a flood of realtime questions from the public. 😄

Pending what the other maintainers think, I'm up for directing people to ask on Twitter instead. I'm pretty active there.

```suggestion
- Publicly tweet [@tseslint on Twitter](https://twitter.com/tseslint).
```